docs/source/flow-project.rst

```diff
@@ -4,29 +4,29 @@
 The FlowProject
 ===============
 
-This chapter describes how to setup a complete workflow via the implementation of a :py:class:`~flow.FlowProject`.
+This chapter describes how to setup a complete workflow via the implementation of a :py:class:`~flow.FlowProject`. It includes two **fundamental concepts** for the implementation of workflows with the **signac-flow** package: :ref:`operations <operations>` and :ref:`conditions <conditions>`.
```

Comment:
```suggestion
This chapter describes how to setup a complete workflow via the implementation of a :py:class:`~flow.FlowProject`.
It includes two **fundamental concepts** for the implementation of workflows with the **signac-flow** package: :ref:`operations <operations>` and :ref:`conditions <conditions>`.
```

Reply:
I think as long as operations and conditions appear as sub-entries in the sidebar toc, we don't need to announce them in the organizational statement (the second sentence).

This sentence makes it sound like there is a central list of "fundamental concepts", which is perhaps another motivation for having a glossary.